src/frontend/apps/standalone_site/src/features/Playlist/components/PlaylistForm.tsx

```diff
@@ -384,7 +389,14 @@ export const PlaylistForm = ({
             />
           )}
           {playlistId && (
-            <Modal isOpen={isModalOpen}>
+            <Modal isOpen={isModalOpen} onClose={() => setIsModalOpen(false)}>
+              <Heading
+                size="3"
+                alignSelf="center"
+                margin={{ top: '0', bottom: 'small' }}
+              >
+                {intl.formatMessage(messages.deleteModalTitle)}
```

Comment:
```suggestion
                {intl.formatMessage(messages.DeleteButtonText)}
```

Reply:
I'm wondering why you put `Delete resource` instead of `Delete playlist`, your button is labeled `Delete playlist`, the modal to create a playlist is labeled as well `Create a playlist` ?

Reply:
I chose to put "Delete resource" in order to keep a similarity with the other modals, and particularly the one used in the `Delete Video` component. The `Delete Video` component is used for both Live and VOD and uses "Delete resource" as wording in it to keep simple the fact that the object we want to delete can be either a recorded video or a live session.

And then I use a more explicit ID just to keep its role clear in the code.